src/styles/_global-styles.scss

```diff
@@ -268,28 +269,125 @@ ul.dso-edit-menu-dropdown > li .nav-item.nav-link {
   vertical-align: middle;
 }
 
-.pt-0\.5 {
-  padding-top: 0.125rem !important;
-}
 
-.pr-0\.5 {
-  padding-right: 0.125rem !important;
-}
 
-.pb-0\.5 {
-  padding-bottom: 0.125rem !important;
+.btn.btn-success{
+	background-color: var(--ds-button-success-background-color);
+	border-color: var(--ds-button-success-background-color);
+	&:hover{
+		background-color: var(--ds-button-success-background-hover-color);
+		border-color: var(--ds-button-success-background-hover-color);
+	}
+}
+.btn.btn-outline-success{
+	border-color: var(--ds-button-success-background-color);
+	color: var(--ds-button-success-background-color);
+	&:hover{
+		background-color: var(--ds-button-success-background-hover-color);
+		color: $white;
+	}
+}
+.btn.btn-outline-warning{
+	border-color:var(--ds-button-warning-background-color);
+	color:var(--ds-button-warning-background-color);
+	&:hover{
+		background-color:var(--ds-button-warning-background-hover-color);
+		color: $white;
+	}
+	&:disabled{
+		background-color: transparent;
+		&:hover{
+			color:var(--ds-button-warning-background-color);
+		}
+	}
+}
+
+
+.btn:focus, .btn-outline-primary:not(:disabled):not(.disabled):active:focus, .btn-outline-primary:not(:disabled):not(.disabled).active:focus, .show > .btn-outline-primary.dropdown-toggle:focus, .custom-control-input:focus ~ .custom-control-label::before, .form-control:focus, .page-link:focus{
+	box-shadow: inset 0 3px 5px rgba(0, 0, 0, 0.25), 0 0 0 0.2rem rgba(27, 41, 55, 0.5);
+}
+
+.btn.btn-success:focus, .btn.btn-outline-success:focus{
+    outline: 2px solid rgba(76, 145, 76, 1);
+    outline-offset: 2px;
+    box-shadow: none;
+}
+.btn.btn-danger:focus, .btn.btn-outline-danger:focus{
+	outline: 2px solid rgba(205, 126, 126, 1);
+    outline-offset: 2px;
+    box-shadow: none;
+}
+.btn.btn-warning:focus, .btn.btn-outline-warning:focus {
+    outline: 2px solid rgba(88, 87, 65, 1);
+    outline-offset: 2px;
+    box-shadow: none;
+}
+.btn.btn-primary:focus, .btn.btn-outline-primary:focus {
+    box-shadow: none;
+    outline: 2px solid rgba(134, 137, 139, 1);
+    outline-offset: 2px;
+}
+.btn.btn-secondary:focus, .btn.btn-outline-secondary:focus {
+    box-shadow: none;
+    outline: 2px solid rgba(141, 148, 155, 1);
+    outline-offset: 2px;
+}
+.btn.btn-warning{
+	background-color:var(--ds-button-warning-background-color);
+	&:hover{
+		background-color:var(--ds-button-warning-background-hover-color);
+	}
+	&:disabled{
+		background-color: transparent;
+	}
+}
+ds-dso-edit-metadata-value {
```

Comment:
Styles that are specific to an Angular component should be moved into the Component's stylesheet.  In this case, you seem to be changing styles for the "dso-edit-metadata-value" component, but you are making those styles difficult for others to find/change by putting them in the global stylesheet (`_global-styles.scss`)

In this situation, I think these styles should move to the stylesheet at `/src/app/dso-shared/dso-edit-metadata/dso-edit-metadata-value/dso-edit-metadata-value.component.scss`

Reply:
I see. I will keep that in mind next time. I moved the component-specific styles to their respective component `.scss` files as you requested.